examples/kitchen-sink/packages/logger/package.json

```diff
@@ -15,14 +15,15 @@
     "test": "jest"
   },
   "jest": {
-    "preset": "jest-presets/jest/node"
+    "preset": "@repo/jest-presets/jest/node"
   },
   "devDependencies": {
-    "@types/node": "^18.11.9",
-    "eslint-config-custom": "workspace:*",
-    "jest-presets": "workspace:*",
-    "tsconfig": "workspace:*",
-    "tsup": "^6.0.1",
-    "typescript": "^5.2.2"
+    "@types/jest": "^29.5.10",
+    "@types/node": "^20.10.0",
+    "@repo/eslint-config": "workspace:*",
+    "@repo/jest-presets": "workspace:*",
+    "@repo/typescript-config": "workspace:*",
+    "tsup": "^8.0.1",
```

Comment:
Should we get rid of this if we're going to follow the internal package pattern?

Reply:
My thinking is to leave in an example of a compiled package while we're still evaluating for future iterations of how we look for how the examples can serve the community best.

There are still a few of these sprinkled around that I've left that way strategically. It's a good callout and a good question to take under consideration going forward.